awx/api/serializers.py

```diff
@@ -1311,6 +1311,8 @@ def validate(self, attrs):
             attrs.pop('local_path', None)
         if 'local_path' in attrs and attrs['local_path'] not in valid_local_paths:
             errors['local_path'] = _('This path is already being used by another manual project.')
+        if attrs.get('scm_refspec') and scm_type != 'git':
+            errors['scm_refspec'] = _('SCM refspec can only be used with git projects.')
```

Comment:
@wenottingham is this a requirement you're comfortable with?  I know we were all quite skeptical about SVN.

Reply:
Yes, I'm OK with that.